src/openzaak/components/zaken/api/filters.py

```diff
@@ -258,6 +259,20 @@ class ZaakFilter(FilterSetWithGroups):
             "gelijk is aan de aangegeven aanduiding worden teruggeven als resultaten."
         ),
     )
+
+    kenmerk__bron = filters.CharFilter(
+        field_name="zaakkenmerk__bron",
+        help_text=mark_experimental(get_help_text("zaken.ZaakKenmerk", "bron")),
+    )
+
+    kenmerk = KeyValueFilter(
+        key_field_name="zaakkenmerk__bron",
+        value_field_name="zaakkenmerk__kenmerk",
+        help_text=mark_experimental(
+            "Een bron-kenmerk combinatie van de zaak. format: [<bron>:<kenmerk>]"
```

Comment:
Do we need the square brackets for these values? I think `[]` are reserved characters in URLs, so they might have to be encoded for it to work.

Locally this seems to work without square brackets as well actually, so can't we just drop that from the format?

```suggestion
            "Een bron-kenmerk combinatie van de zaak. format: `<bron>:<kenmerk>`"
```

Reply:
The validator didn't work, i removed the brackets around the k:v but should the brackets still be disallowed in the key & value themselves? or maybe only allow `[a-z][A-Z]`, `_` & `-`?

Reply:
add didgets for the value

Reply:
`[a-zA-Z0-9_-]+` but then the db fields should also have this as validation.